server/sonar-web/src/main/js/apps/tutorials/onboarding/commands/Msvc.js

```diff
@@ -33,19 +33,22 @@ type Props = {|
 |};
 */
 
+// onboarding.analysis.sq_scanner.docs
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ACommentedCode)